splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/RowIdIT.java

```diff
@@ -450,4 +451,30 @@ public void testRowIdToInstantFunctionInvalidInput() {
         }
         fail("Expected: java.sql.SQLException: ERROR 22008: '33445567' is an invalid argument to the TO_INSTANT function.");
     }
+
+    @Test
+    public void testRowIdToHbaseEscapedFunction() throws Exception {
+        ResultSet resultSet  = methodWatcher.executeQuery(String.format("select cast(rowid as varchar(30)) from %s",
+                this.getTableReference(TABLE7_NAME)));
+        Assert.assertTrue(resultSet.next());
+        String rowId = resultSet.getString(1);
+        Assert.assertFalse(resultSet.next());
+        StringDataValue result = RowIdUtil.toHBaseEscaped(new SQLChar(rowId));
+        resultSet = methodWatcher.executeQuery(String.format("select to_hbase_escaped(rowid) from %s", this.getTableReference(TABLE7_NAME)));
+        Assert.assertTrue(resultSet.next());
+        Assert.assertEquals(result.getString(), resultSet.getString(1));
```

Comment:
this looks like you're comparing the result against the result.
maybe instead of result use expectedEscaped.